### DISCUSSION TITLE
List of the best dates

### DISCUSSION THREAD

**Discussion by h0jeZvgoxFepBQ2C** [lukevella/rallly#284] [Ideas]
It's getting quite complicated if you offer f.e. 30 possible dates to select and want to find out which is the best date in the end.. 
So it would be nice to see f.e. a top 10 list, where the best dates (most acceptions) are listed?

**Comment by lukevella** [comment]
Yes, this has been on my radar for a while. Can't give an ETA right now because I'm working on a major update and this will come after that but I will definitely work on a solution for this.

**Reply by h0jeZvgoxFepBQ2C** [reply]
Cool, great to hear! Is there a list which features will come in the next major update? Thank you so much for your great work! ❤️

**Reply by lukevella** [reply]
I haven't published anything yet. The initial release will just have updated navigation and layout and that will be setting the foundation for new features to be added.
